### ISSUE TITLE
Cannot connect to server build from 'master' branch, only 'michelle-express-rewrite' branch works

### ISSUE THREAD

**Issue by madeofsun** [peers/peerjs-server#122]
When I use code from the 'master' branch, request for id results in 404 error, also there is CORS problem. On the other hand, 'michelle-express-rewrite' branch works fine without any of this problems. Same for both: localhost and my cloud machine.
Maybe I setting it up somehow wrong, what can it possibly be?

I use peer.min.js 1.0.0 as client

**Comment by kidandcat** [comment]
I don't say we shouldn't send CORS header. But the recommendation is always to expose your apps though a frontline server, let me recommend you Caddy:

https://caddyserver.com/

The configuration is as easy as:
(this is our PeerServer config)

```
0.peerjs.com {
  proxy / http://localhost:9000 {
    websocket
  }
  cors / {
    origin            *
    methods           POST,PUT,GET
    allowed_headers   Content-Length,range
    exposed_headers   Content-Length,range
  }
  tls kidandcat@gmail.com
}
```

Port 9000 is open in 0.peerjs.com for backward compatibility, but you should keep it closed, or expose PeerServer only in 127.0.0.1:9000

**Comment by madeofsun** [comment]
@kidandcat 
CORS it not the main problem. Even if I get rid of it, it is still 404 error on the initial request for id
`/peerjs/id?ts=15561101165850.8761715781518726` 

I tried your your configuration with Caddy, but it is still the same

**Comment by kidandcat** [comment]
Have you configured a different key for peerserver?

**Comment by madeofsun** [comment]
I run 
`./bin/peerjs --port 9000`

And then connect with
`const peer = new Peer({host: 'localhost', port: 9000})`

**Comment by afrokick** [comment]
> Anyway @afrokick we just need to:
> ```
> const cors = require('cors')
> const app = express()
> app.use(cors())
> ```
> Could you take care of it?

It is already exists https://github.com/peers/peerjs-server/blob/master/src/api/index.js#L13

**Comment by lagcamgc** [comment]
Hello, i have a similar issue, but mine is that it does not detects connection for the version on the 'master' branch but in the branch 'michelle-express-rewrite' it works, i.e. the connection does not open or detect an incoming connection but in that metioned branch it works, i test it locally to be sure that i don't have a nginx problem or something like that, also in your servers it works, are you guys running the master branch there?

**Comment by afrokick** [comment]
Probably fixed in #125
